client/extensions/woocommerce/app/order/order-customer/index.js

```diff
@@ -157,11 +178,16 @@ export default connect(
 		const isEditing = isCurrentlyEditingOrder( state );
 		const order = isEditing ? getOrderWithEdits( state ) : getOrder( state, props.orderId );
 
+		const loadedLocations = areLocationsLoaded( state, siteId );
+		const countries = getAllCountries( state, siteId );
+
 		return {
+			countries,
```

Comment:
p'typez

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/22838/commits/705b284462b5390c6fa399f6c7eaef08f0cc96a9